src/v/pandaproxy/schema_registry/avro.cc

```diff
@@ -509,21 +520,25 @@ sanitize_avro_schema_definition(unparsed_schema_definition def) {
     sanitize_context ctx{.alloc = doc.GetAllocator()};
     auto res = sanitize(doc, ctx);
     if (res.has_error()) {
+        // TODO BP: Prevent this linearizaton
```

Comment:
chore: Do we need a follow up issue for this?

Reply:
Yeah, I'll add one